department-of-veterans-affairs/abd-vro
# Chengjie8/issue 2033

## What was the problem?
per [slack instructions](https://dsva.slack.com/archives/C03UBMAP5JQ/p1694722111022789?thread_ts=1694721919.435159&cid=C03UBMAP5JQ) to add a new github workflow for dora lead time

Associated tickets or Slack threads:
- #2033 



## How to test this PR
- Run the workflow


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
